starstream-compiler/src/parser/snapshots/starstream_compiler__parser__statement__tests__block.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: starstream-compiler/src/parser/statement.rs
+assertion_line: 186
```

Comment:
Not sure what's up with these lines appearing and disappearing between you and me. Version mismatch? OS difference?